miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/PrinterService.java

```diff
@@ -21,28 +21,35 @@
  * *********************************************************************
  */
 
-package uk.ac.bbsrc.tgac.miso.core.service.printing.context;
+package uk.ac.bbsrc.tgac.miso.service;
 
 import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
 
-import net.sourceforge.fluxion.spi.Spi;
+import uk.ac.bbsrc.tgac.miso.core.data.Printer;
 
 /**
- * uk.ac.bbsrc.tgac.miso.core.service.printing.context
+ * uk.ac.bbsrc.tgac.miso.core.manager
  * <p/>
- * Info
+ * This interface describes a manager that can discover and print to javax.print PrintServices
```

Comment:
Remove or update Javadoc - This is used for retrieving and modifying Printers - not discovery or printing